pure.zsh

```diff
@@ -517,7 +517,9 @@ prompt_pure_state_setup() {
 		who_out=$(who -m 2>/dev/null)
 		if (( $? )); then
 			# Who am I not supported, fallback to plain who.
-			who_out=$(who 2>/dev/null | grep ${TTY#/dev/})
+			declare -a who_in
```

Comment:
This is a bit pedantic but could we change `declare` to `local` here? That, or `typeset`. The reason I'm suggesting it is that Pure doesn't use `declare` anywhere else, and for anyone reading this code, the mixture of declarations might just cause confusion. I.e. why use `declare` here, does it have a special purpose? I personally like `local` due to the additional information the name conveys (this is a local variable) albeit using `typeset` (or `declare` for that matter) in this context produces the exact same result.

Apologies for the wall of text 😅.

Reply:
Whoops, I thought there was something going on with the `local` keyword and `-a`, but apparently I misread `-g` as `-a` in the manpage.  Committing.